modules/headless_ai/cfgFunctions/GetInfo/fn_getDetailsGroup.sqf

```diff
@@ -34,14 +34,13 @@ GETVAR(_group,TaskTimer,0),
 GETVAR(_group,multiplier,0),
 _occupy,
 _vehAssigned,
-([(waypoints _unit)] call FUNC(getWaypointDetails)),
-(surfaceIsWater (getposATL _unit)),
+[waypoints _unit] call FUNC(getWaypointDetails),
+surfaceIsWater getposATL _unit,
```

Comment:
For me at least, the brackets around `getposATL _unit` make the intention clearer, just a personal preference.